slackbridge/bots.py

```diff
@@ -108,4 +111,17 @@ def signedOn(self):
             self.join('#{}'.format(channel['name']))
 
     def post_to_irc(self, channel, message):
-        self.msg(channel, message)
+        self.msg(channel, self.__format_message(message))
+
+    def __format_message(self, message):
+        match_ids = re.findall(r'(<\@([A-Z0-9]{9,})\>)', message)
+        # If for some weird reason someone mentions the same person
+        # twice in one message, we avoid extra api calls
```

Comment:
There aren't even any API calls here any more, so I believe this comment should be updated